cmd/xenia/web/web.go

```diff
@@ -4,50 +4,99 @@ import (
 	"fmt"
 	"io"
 	"io/ioutil"
+	"log"
 	"net/http"
+	"os"
 
 	"github.com/ardanlabs/kit/cfg"
+	"github.com/coralproject/shelf/internal/platform/auth"
 	"github.com/spf13/cobra"
 )
 
 const (
-	cfgHost = "WEB_HOST"
-	cfgAuth = "WEB_AUTH"
+	// cfgWebHost is the url to the web host which includes the hostname, port,
```

Comment:
I like whitespace before every comment, and I think Bill is using this convention.